README.md

```diff
@@ -41,13 +40,50 @@ The `out` option can also be an array of files:
 }
 ```
 
+## application/json & text/yaml
+
 The `path` option (default: `"version"`) can be used to change a different property. the following example will set the
-`deeper.current` property to the new version in `manifest.json`:
+`deeper.current` property to the new version in `manifest.json` and `info.yml`:
+
+```json
+"plugins": {
+  "@release-it/bumper": {
+    "out": { "file": "manifest.json", "path": "deeper.current" },
+    "out": { "file": "info.yml", "path": "deeper.current" }
+  }
+}
+```
+
+## text/plain
+
+Two modes are supported :
+
+### Whole file
+
+In that case the whole file is used to read and/or write the version.
+
+It's the default and the only option for `in` files.
 
 ```json
 "plugins": {
   "@release-it/bumper": {
-    "out": { "file": "manifest.json", "path": "deeper.current" }
+    "in": { "file": "VERSION", "type": "text/plain" },
+    "out": { "file": "VERSION", "type": "text/plain" }
   }
 }
 ```
+
+### Replacements
+
+Using the special path `replace` for `text/plain` files, the last version number is replaced by the new one.
```

Comment:
I'm not sure about a better alternative, but using "replace" for `path` doesn't feel exactly right (in theory it even conflicts with actual intention of using this prop literally). Maybe something like a new attribute "mode" (maybe "global"?). But really, better suggestions are most welcome.

Reply:
You're right, for cleaner and clearer options, it's better to use a separate option.  
As it's a feature which is on/off (replace entire file content or parts), I suggest something like mode (parts | full) or regex (true | false).